### ISSUE TITLE
Training from Scratch

### ISSUE THREAD

**Issue by melihyazgan** [ylwhxht/V2X-R#16]
Hi,
Thank you for your effort on this great codebase.
Is it possible to share an image of Validation Loss or/and Training Loss from Tensorboard? I am testing mdd model on another dataset and have deactivated your FOV to utilize more radar (360°). Right now, the diffusion loss is dominating the total loss.

Br,
Melih
